arclightning_backend/src/splash.rs

```diff
@@ -0,0 +1,16 @@
+extern crate rand;
```

Comment:
This should be in `arclightning_backend/src/main.rs`
Also, add `mod splash` to `arclightning_backend/src/main.rs`

Reply:
After moving the extern crate, you should add `use rand` here instead.

Reply:
After this, you should be able to compile your code (`cargo build`) and see if it compiles